keyboards/launchpad/readme.md

```diff
@@ -27,6 +27,16 @@ See [build environment setup](https://docs.qmk.fm/#/getting_started_build_tools)
 
 -=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-
 
+![Underglow](https://i.imgur.com/3zFIOmu.jpg)
```

Comment:
This info should also be included in a `keymaps/default_rgb/readme.md`

Reply:
Done!